crates/ide-completion/src/render/function.rs

```diff
@@ -701,40 +719,52 @@ fn g(foo: (), mut bar: u32, baz: ())
     }
 
     #[test]
-    fn complete_fn_mut_param_has_attribute() {
```

Comment:
Did you remove this test?

Reply:
Oops I thought I copied it and adjusted it for the new test... Guess I only adjusted it :grin: